libdnf/transaction/Swdb.cpp

```diff
@@ -54,8 +54,13 @@ Swdb::Swdb(const std::string &path)
   : conn(nullptr)
   , autoClose(true)
 {
+    // check if writing to an in-memory database
+    if (path == ":memory:") {
+        conn = std::make_shared< SQLite3 >(path);
+        Transformer::createDatabase(conn);
+    }
     // check if DB file is present
-    if (!pathExists(path.c_str())) {
+    else if (!pathExists(path.c_str())) {
```

Comment:
Minor: this formatting looks odd to me. Let's just put it on the same line as the previous closing brace?

Reply:
I tweaked the comments a little bit, it's hopefully better now.